README.md

```diff
@@ -128,7 +128,7 @@ The following are valid query attributes:
 | `id` | The unique ID that is associated with each Dataset |
 | `keyword` | The topic of the Dataset |
 | `modified` | The date the File was last modified |
-| `resource_uri` | Location of the File |
+| `irm_url` | Location of the IRM document |
```

Comment:
:question:  Should URLs go in this list?

Reply:
I'm not sure if it makes sense to query on a URL.